modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ActivityUI.es.js

```diff
@@ -0,0 +1,234 @@
+};
+
+const getQuestionCreatedInDays = (dateCreated) => {
+	const givenDate = new Date(dateCreated);
+	const now = new Date();
+	const timeDifference = now.getTime() - givenDate.getTime();
+	const diffDays = (timeDifference / TIME_IN_DAYS).toFixed(0);
+
+	return diffDays;
+};
+
+const ActivityHeaderBadge = ({badgeField: {label, symbol, type}, question}) => {
+	const DAYS_SINCE_CREATED = getQuestionCreatedInDays(question.dateCreated);
+
+	return (
+		<div className="align-items-center d-flex flex-wrap justify-content-between">
+			<ul className="align-items-center c-mb-2 d-flex flex-nowrap list-badges list-unstyled stretched-link-layer">
+				{DAYS_SINCE_CREATED <= DAYS_UNTIL_SHOW_LABEL &&
+					type !== MESSAGE_TYPES.reply.type && (
+						<li>
+							<span className="new-question-badge text-uppercase">
+								{Liferay.Language.get('new')}
+							</span>
+						</li>
+					)}
+
+				<li>
+					<QuestionBadge
+						className={classNames(
+							'bg-light label-secondary text-uppercase',
+							{
+								'questions-reply': symbol === 'reply',
+							}
+						)}
+						iconInvert={classNames({
+							'questions-reply-icon': symbol === 'reply',
+						})}
+						isActivityBadge
+						symbol={symbol}
+						value={label}
+					/>
+				</li>
+
+				<li>
+					<QuestionBadge
+						className="bg-light label-secondary text-uppercase"
+						isActivityBadge
+						value={question.messageBoardSection.title}
+					/>
+				</li>
+			</ul>
+		</div>
+	);
+};
+
+const ActivityHeader = ({
+	badgeField: {text},
+	context,
+	question: {id, locked, seen, status},
+}) => (
+	<h5
+		className={classNames(
+			'questions-labels-limit',
+			'stretched-link',
+			'c-mb-0',
+			'stretched-link-layer',
+			'text-dark',
+			{
+				'question-seen': seen || context?.Visited?.includes(id),
+			}
+		)}
+	>
+		{text}
+
+		{status && status !== 'approved' && (
+			<span className="c-ml-2">
+				<ClayLabel displayType="info">{status}</ClayLabel>
+			</span>
+		)}
+
+		{!!locked && (
+			<span className="c-ml-2">
+				<ClayIcon
+					data-tooltip-align="top"
+					symbol="lock"
+					title={Liferay.Language.get(
+						'this-question-is-closed-new-answers-and-comments-are-disabled'
+					)}
+				/>
+			</span>
+		)}
+	</h5>
+);
+
+const ActivityBody = ({badgeField: {symbol, type}, question}) => {
+	if (type === MESSAGE_TYPES.answer.type) {
+		return (
+			<ArticleBodyRenderer
+				{...question}
+				articleBody={stripHTML(question.articleBody)}
+				compactMode
+			/>
+		);
+	}
+
+	if (type === MESSAGE_TYPES.reply.type) {
+		return (
+			<>
+				<ArticleBodyAnwser
+					{...question}
+					articleBody={stripHTML(question.articleBody)}
+					compactMode
+				/>
+
+				<QuestionBadge
+					className="questions-reply"
+					iconInvert="questions-comment-reply-icon"
+					isActivityBadge
+					symbol={symbol}
+					value={stripHTML(question.articleBody)}
+				/>
+			</>
+		);
+	}
+	if (type === MESSAGE_TYPES.question.type) {
+		return null;
+	}
```

Comment:
I think return `null` Is enough

```suggestion

	return null;
```